tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -410,7 +386,10 @@ Validate line chart embeds correctly
 Configure basic vertical bar chart
     [Tags]  HappyPath
     user goes to url  ${DATABLOCK_URL}
-
+    user configures basic vertical bar chart   ${DATABLOCK_NAME}  500  800
+    
+Change vertical bar chart legends
```

Comment:
This test case can be removed as `user configures basic vertical bar chart` covers it